scout/server/blueprints/clinvar/templates/clinvar/multistep_add_variant.html

```diff
@@ -33,7 +33,7 @@
                    <li>Observation Data</li>
               </ul>
               <!-- fieldsets -->
-              <fieldset>
+              <fieldset data-step="1">
```

Comment:
The problem is that in general forms are validated at the end, when you submit. But we want to be able to validate every single step before passing to the next. This way you can do it when the user presses "next"